src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
-
+//import Navigation from './components/Navigation';
+import {Route} from "react-router-dom";
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path='/' component={Home}/>
+    <Route exact path='/' component={About}/>
```

Comment:
Do you want all your paths to be the same?

Reply:
"/" is the typical path for Home or Landing pages since it is the first page of the application, but you may want about and contact to have different urls.